### PR TITLE
Fix ckpt conversion bug

### PR DESCRIPTION
**Pull request by zhr1201**
A bug was introduced in https://github.com/wenet-e2e/wenet/pull/2381, basically it will skip copying all the weight if remove_list is empty.

### CODE REVIEW

File: examples/aishell/whisper/local/modify_ckpt.py [hunk 1]
```diff
@@ -19,14 +19,16 @@
...
     parser = argparse.ArgumentParser(description='filter out unused module')
-    parser.add_argument('--remove_list',
-                        default="",
-                        type=str,
-                        help='list of name filter, comma-separated, e.g."name1, name2"')
-    parser.add_argument('--add_list',
-                        default="",
-                        type=str,
-                        help='dict of name adder, e.g."{\"key1\": \"value1\"}"')
+    parser.add_argument(
```
Comment (zhr1201) [inline examples/aishell/whisper/local/modify_ckpt.py:22]: yapf was complaining

File: examples/aishell/whisper/local/modify_ckpt.py [hunk 2]
```diff
@@ -39,18 +41,22 @@ def main():
 
     state = torch.load(args.input_ckpt, map_location="cpu")
     new_state = {}
+
     if args.remove_list:
```
Comment (zhr1201) [inline examples/aishell/whisper/local/modify_ckpt.py:45]: it skips cloning the weight if remove_list is empty

File: wenet/transformer/search.py [hunk 3]
```diff
@@ -175,7 +175,7 @@ def ctc_prefix_beam_search(
                         next_score1.ns = log_add(next_score1.ns,
                                                  prefix_score.ns + prob)
                         if next_score1.v_ns < prefix_score.v_ns + prob:
-                            next_score1.vs_ns = prefix_score.v_ns + prob
+                            next_score1.v_ns = prefix_score.v_ns + prob
```
Comment (zhr1201) [inline wenet/transformer/search.py:178]: typo

**Comment by zhr1201** [comment]
there seems to be a clang format version mismatch in pre commit hook and github workflow

**Comment by xingchensong** [comment]
https://github.com/wenet-e2e/wenet/pull/2398

**Comment by zhr1201** [comment]
pulled in the version fix from main

**Review by xingchensong — APPROVED** [review]
thx

**Pull request merged**